.github/workflows/cypress.yaml

```diff
@@ -163,14 +163,17 @@ jobs:
                   headed: true
                   start: npx serve -p 8080 -L ../webapp
                   wait-on: "http://localhost:8080"
-                  record: true
+                  record: false
                   parallel: false
-                  command-prefix: "yarn percy exec --parallel --"
+                  command: npx knapsack-pro-cypress --config trashAssetsBeforeRuns=false --browser chrome
```

Comment:
This specifying `--browser chrome` implies it doesn't respect the `browser` field above L162